app/assets/javascripts/editor.js

```diff
@@ -1,6 +1,89 @@
+/* global CodeMirror */
+/* eslint-disable no-alert */
+
 +function () {
+  var urlQueryParams = function () {
+    var params     = {}
+    var paramsList = (location.search || '').substr(1).split('&')
+
+    for (var i = 0; i < paramsList.length; ++i) {
+      // :pray: for the Uganda children
+      var param = paramsList[i].split('=', 2)
+
+
+      if (param.length == 2)
+        params[param[0]] = decodeURIComponent(param[1].replace(/\+/g, ' '))
+    }
+
+    return params
+  }
+
+  var jumpToLine = function (editor, $textarea) {
+    var params = urlQueryParams()
+
+    if (params['line']) {
+      var line = +params['line'] - 1
+
+      if (params['model_id'] && params['model_id'] != $textarea.data('modelId')) {
+        return
+      }
+
+      editor.scrollIntoView({ line: line, ch: 0 }, 200)
+      editor.setCursor({ line: line, ch: 0 })
+      editor.focus()
+    }
+  }
+
+  var startEditors = function () {
+    $('[data-editor]:not([data-observed])').each(function (i, element) {
+      var $textarea = $(element)
+      var $file     = $($textarea.data('fileInput'))
+      var $change   = $($textarea.data('changeInput'))
+      var readonly  = $textarea.data('readonly')
+      var editor    = CodeMirror.fromTextArea($textarea.get(0), {
+        mode:              'ruby',
+        tabSize:           2,
+        autoCloseBrackets: true,
+        matchBrackets:     true,
+        lineNumbers:       true,
+        styleActiveLine:   true,
+        foldGutter:        true,
+        theme:             readonly ? 'solarized dark' :  'solarized light',
```

Comment:
Esto es gravísimo =P

```suggestion
        theme:             readonly ? 'solarized dark' : 'solarized light',
```